src/guidescanpy/commands/add_organism.py

```diff
@@ -11,17 +12,34 @@
 
 def insert_chromosome(organism, chr2acc_file):
     result = []
-    with open(
-        chr2acc_file,
-    ) as f:
-        reader = csv.DictReader(f, delimiter="\t")
+    with open(chr2acc_file) as infile, open("temp", "w", newline="") as outfile:
+        reader = csv.DictReader(infile, delimiter="\t")
+        new_header = [
+            field.lower().replace("# ", "").replace("#", "")
+            for field in reader.fieldnames
+        ]  # format csv file header
+        reader.fieldnames = new_header
+
+        writer = csv.DictWriter(outfile, fieldnames=new_header, delimiter="\t")
+        writer.writeheader()  # write the chr2acc file with formatted header
+
         for row in reader:
+            writer.writerow(row)
+            if "ucsc" in row:
+                accession = row["refseq"]
+                name = row["ucsc"]
+            else:
+                accession = row["accession.version"]
+                name = row["chromosome"]
+            if name.startswith("chr"):
+                name = name[3:]
             insert_chromosome_query(
-                accession=row["Accession.version"],
-                name=row["#Chromosome"],
+                accession=accession,
+                name=name,
                 organism=organism,
             )
-            result.append(row["Accession.version"])
+            result.append(accession)
+    os.rename("temp", chr2acc_file)
```

Comment:
This is kind of a dangerous thing to do - a function that is supposedly reading a file should not overwrite the file from under the caller's feet.